src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/response/SearchStaffUserByIdResponse.java

```diff
@@ -1,33 +1,17 @@
 package uk.gov.hmcts.reform.cwrdapi.controllers.response;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
-import uk.gov.hmcts.reform.cwrdapi.client.domain.Location;
-import uk.gov.hmcts.reform.cwrdapi.client.domain.Role;
-import uk.gov.hmcts.reform.cwrdapi.client.domain.ServiceResponse;
-import uk.gov.hmcts.reform.cwrdapi.client.domain.SkillResponse;
-
-import java.util.List;
+import lombok.experimental.SuperBuilder;
```

Comment:
its not recommended to use experimental features, find alternative solution to achieve the same.

Reply:
done, Removed